bgpd/bgp_route.c

```diff
@@ -10931,10 +10934,12 @@ DEFUN (show_ip_bgp_json,
 		else
 			return (bgp_show(vty, bgp, afi, safi,
 					 bgp_show_type_community_all, NULL,
-					 uj));
+					 uj, 0 /*wide*/));
 	}
 
-	return bgp_show(vty, bgp, afi, safi, sh_type, NULL, uj);
+	bool wide = argv_find(argv, argc, "wide", &idx);
```

Comment:
declare bool wide at the top with all the other variables

Reply:
addressed in the latest commit